src/main/java/edu/ucsb/ucsbcslas/controllers/OnlineOfficeHoursController.java

```diff
@@ -40,6 +63,14 @@
     private OnlineOfficeHoursRepository officeHoursRepository;
     @Autowired
     private TutorRepository tutorRepository;
+    //new
```

Comment:
This comment will lose its meaning once it is merged into main, as it is no longer "new", please remove.